### ISSUE TITLE
python-cloudflare is not PEP-561 aware

### ISSUE THREAD

**Issue by chris-zenfolio** [cloudflare/python-cloudflare#174]
I am writing Python code for some automation I am creating, but when testing my code with mypy, I always get the following error:
```
error: Skipping analyzing "CloudFlare": module is installed, but missing library stubs or py.typed marker  [import-untyped]
```
This error is generated when mypy hits the following line:
```
import Cloudflare
```

Please make python-cloudflare [PEP-561](https://peps.python.org/pep-0561/) aware.
Here is a link to the [mypy docs relevant to this error](https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-library-stubs-or-py-typed-marker).

**Comment by mahtin** [comment]
Thanks for pointer to PEP-561. Looks like a useful thing. Feel free to PR the required changes or wait for me to free-up some time.

**Comment by chris-zenfolio** [comment]
@mahtin : I looked at what it would take, but I don't understand the code well enough to create a PR for this.  I spent over an hour last night trying to figure out what data type would be returned from a single call to get a Zone and then Python threw up all over me because my type hints in my functions were never right.  I think it has to do with how API calls are wrapped?  (eg `cloudflare.zones.dns_records.post()`)
